CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
 - Added simulation function to `hf iclass legrec` (@antiklesys)
 - Added keys from Momentum firmware projects. (@onovy)
 - Added Dutch Statistics Agency default key (@eagle00789)
+- Added cuxtom suffix for `hf mf autopwn` command. (@zxkmm)
```

Comment:
Change to: 

- Changed `hf mf autopwn`  - now allows for custom suffix (@zxkmm)